File: python/pangram/pangram.py
# This method accepts a string called sentence and returns a bool (T/F)
def is_pangram(sentence: str) -> bool:
    #makes the string all lowercase
    sentence = sentence.lower()
    #a string containing all letters of the alphabet
    letters = 'abcdefghijklmnopqrstuvwxyz'
    #If the variable sentence has all letter contained in string it returns true
    return set(sentence).issuperset(letters)

File: php/pangram/pangram.php
<?php
function isPangram(string $input) {
 
    //gives an array containing all letters a-z
    $letters = range('a', 'z');
    //splits each input character into an array index for
    $input = str_split(strtolower($input));
    
    //If the letters array and input array contain 26 intersections (input has a value for each letter) then return true 
    
    return count(array_intersect($letters,$input)) == 26 ? true:false;
    
}